src/modules/udw/common/selected-content/selected.content.component.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import SelectedContentItemComponent from './selected.content.item.component';
+import PopupComponent from '../../../common/tooltip-popup/tooltip.popup.component';
+import { createCssClassNames } from '../../../common/css-class-names/css.class.names';
+
+const noConfirmedContentTitle = Translator.trans(
```

Comment:
some variables holding translations text use UPPER_CASE, and here we have camelCase, is this intended? :)

Reply:
Good catch. I'll correct that here and there.